docs/userguides/contracts.md

```diff
@@ -30,7 +30,7 @@ dev = accounts.load("dev")
 contract = dev.deploy(project.MyContract)
 ```
 
-If your contract requires constructor arguments then you will need to pass them to the contract in the [kwargs](../methoddocs/transactions.html) when deploying like this:
+If your contract requires constructor arguments then you will need to pass them to the contract in the [kwargs](./transactions.html) when deploying like this:
```

Comment:
This seems wrong; the `kwargs` are for `txn` obj properties and the `args` are the arguments to the constructor.

```suggestion
If your contract requires constructor arguments then you will need to pass them to the contract in the [args](./transactions.html) when deploying like this:
```